geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/stats/AccumuloGeoMesaStats.scala

```diff
@@ -79,9 +78,8 @@ class AccumuloGeoMesaStats(ds: AccumuloDataStore, val metadata: AccumuloBackedMe
     StatsCombiner.configure(sft, connector, metadata.table, metadata.typeNameSeparator.toString)
 
     val keys = Seq(CountKey, BoundsKeyPrefix, TopKKeyPrefix, FrequencyKeyPrefix, HistogramKeyPrefix)
-    val splits = keys.map(k => new Text(metadata.encodeRow(sft.getTypeName, k)))
-    // noinspection RedundantCollectionConversion
-    connector.tableOperations().addSplits(metadata.table, ImmutableSortedSet.copyOf(splits.toIterable))
+    val splits = new java.util.TreeSet[Text](keys.map(k => new Text(metadata.encodeRow(sft.getTypeName, k))).asJava)
```

Comment:
The new TreeSet is immutable and/or the previous code that made an immutable copy is unnecessary?

Reply:
the new one is *not* immutable, but i don't think it's necessary. guava offers an immutable one which i assume was used just out of preference for using immutable collections, but it doesn't seem worth requiring guava here just for that.

Reply:
Cool.  Thanks.